### ISSUE TITLE
Test Case  5_5, NaN

### ISSUE THREAD

**Issue by chooweixiang98** [nus-cs2030/2021-s2#311]
# Summary
does anyone know what are the circumstances that would result in NaN for test case 5_5?

## Screenshots (if any): 
![image](https://user-images.githubusercontent.com/26595217/115878531-3ad67d00-a47b-11eb-83c2-572e1fa94a7a.png)

**Comment by random689** [comment]
I believe its cause the number of people served is 0, so the average wait time will be divided by 0 which results in NaN so you need to work around it.

**Comment by hewtungyuen** [comment]
this test case has number of servers = 0, so you could include a condition to check for that

**Comment by ARPspoofing** [comment]
In your toString include an if else if number of people served is 0. CodeCrunch doesn't flag out this check so it should be fine

**Comment by Xeph-o** [comment]
All your customers left and there is probably a division by 0 error since the number of customer served (the denominator) is 0.

**Comment by gracefour11** [comment]
yup, agree with the comments above. Just need do some if else statement to account for this division by 0, i.e. num of ppl served = 0 scenario

**Comment by linminsaw** [comment]
NaN appears because of invalid operations like 0 division error, so you probably need to include a condition to check whether the number of people served is 0 before printing it out.

**Comment by darylyeejh** [comment]
If I'm not mistaken, normally you'll get ArithmeticException which happens if 0 is an integer. However, you will get NaN if you are dividing with 0 with floating points.

**Comment by brianlyc** [comment]
i think what you can do is that you can just have an if else statement in your toString method to print out the line if there are no customers served.

**Comment by zhuoyunhui** [comment]
use an if else statement to detect NaN values, you can check for NaN using Double.isNaN(__)

**Comment by Jason-Ng00** [comment]
I think you can just use an if else statement to check for the case when your devisor (probably the number of customer served) is 0

**Comment by MinErn-Chen** [comment]
can initialise a double var to get the result of the average time, then have an if statement directly below it checking if numServed == 0.  if so, change the var to 0 then change the first argument in ur System.out.println() to var instead :)
